packages/patternfly-4/react-table/src/components/Table/BodyCell.tsx

```diff
@@ -22,8 +22,10 @@ export const BodyCell: React.FunctionComponent<BodyCellProps> = ({
   isVisible,
   parentId,
   textCenter = false,
+  /* eslint-disable @typescript-eslint/no-unused-vars */
   isOpen,
```

Comment:
Is there a reason to keep `isOpen` if it's not used?

Reply:
@dlabrecq thanks for calling this out - I've seen this pattern where we pull certain variables out of props (even if we don't use them) so they're not passed in with the remaining `...props` - I'm not as familiar with tables so didn't know if that was the case here, so simply avoided the existing error by disabling it for these variables.  Do you think the unused variables should be removed instead?  Same issue for your next comment below on SelectColumn.

Reply:
I think due to the way we're spreading props, if we don't destructure these properties in this way, they end up getting passed down to the DOM element which throws warnings in the console like so

`Warning: React does not recognize the `isOpen` prop on a DOM element.`